stuff/javascript/jasmine-standalone-2.8.0/spec/getExponentSpec.js

```diff
@@ -0,0 +1,14 @@
+describe('getExponent', function() {
+    it('should give the exponent', function() {
+        var result = getExponent(4, 2);
+        var expected = 16;
+
+        expect(result).toBe(expected);
+    });
+
+    it('should throw an error on no arguments', function() {
+        expect(function() {
+        	getExponent();
+        }).toThrowError();
+    });
+});
```

Comment:
The message "No new line at end of file" is not a critical problem. But please, check this link -> 

https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file